data/txHandler.go

```diff
@@ -173,21 +193,23 @@ func (handler *TxHandler) backlogWorker() {
 		select {
 		case wi, ok := <-handler.backlogQueue:
 			if !ok {
+				// this is never happening since handler.backlogQueue is never closed
 				return
 			}
 			if handler.checkAlreadyCommitted(wi) {
 				transactionMessagesAlreadyCommitted.Inc(nil)
 				continue
 			}
-
-			// enqueue the task to the verification pool.
-			handler.txVerificationPool.EnqueueBacklog(handler.ctx, handler.asyncVerifySignature, wi, nil)
+			handler.streamVerifierChan <- verify.UnverifiedElement{TxnGroup: wi.unverifiedTxGroup, BacklogMessage: wi}
 
 		case wi, ok := <-handler.postVerificationQueue:
```

Comment:
I don't think it will be any help.
The separation of transactions happens in the exec pool, with the information on how to separate it in the same goroutine. When moved to after postVerificationQueue, that operation will be sequential, adding more work the the backlog worker.